### ISSUE TITLE
xml.etree.cElementTree removed from Python 3.9 in favour of xml.etree.ElementTree

### ISSUE THREAD

**Issue by tirkarthi** [mgedmin/check-manifest#112]
Places : https://github.com/mgedmin/check-manifest/search?q=celementtree&unscoped_q=celementtree

Ref : https://github.com/python/cpython/pull/19108

**Comment by tirkarthi** [comment]
I will raise a PR for this issue.

**Comment by mgedmin** [comment]
Thank you for the heads up!

The last time I tried to run tox on 3.9a3 the main problem was that I'm still using `nose` as the test runner, and nose doesn't work on 3.9 (AttributeError: module 'collections' has no attribute 'Callable').  There was an attempt to switch to pytest (#92), but it foundered on a pytest problem.

It looks like some of the removals have been reverted in 3.9a5, because `tox -e py39` now passes for me.  Ah, I see that I did a 'make install' without wiping the old installed tree, so I still had a cElementTree from the alpha 3.  Doing a fresh install exposes the problem.

Again, thank you!
